app/src/renderer/vuex/store.js

```diff
@@ -83,6 +83,7 @@ export function storeUpdateHandler(mutation, state, pending) {
  */
 function persistState(state) {
   const cachedState = JSON.stringify({
+    //blocks: state.blockchain.blocks,
```

Comment:
why don't we want to cache blocks?

Reply:
hmm yeah not sure. we don't want there to be no blocks, but we also don't want to show old blocks. so probably we don't need the cache.

Reply:
this is the comment i was talking about